pkg/controller/volume/attachdetach/metrics/metrics.go

```diff
@@ -132,3 +146,59 @@ func (collector *volumeInUseCollector) getVolumeInUseCount() nodeVolumeCount {
 	}
 	return nodeVolumeMap
 }
+
+// Check if our collector implements necessary collector interface
```

Comment:
Rather than creating a new collector here - can we not just reuse existing one and possibly rename it, so as it better suites the new name?

Reply:
Done.